main.go

```diff
@@ -231,7 +233,7 @@ func runInTrainingMode(host string, reBaseAll bool, testSuite *testStrategies.Te
 func runInTestingMode(
 		basePerfstats *perfTestUtils.BasePerfStats,
 		host string,
-		frg func(*perfTestUtils.BasePerfStats, *perfTestUtils.PerfStats, *perfTestUtils.Config, perfTestUtils.FileSystem, string),
+		frg func(*perfTestUtils.BasePerfStats, *perfTestUtils.PerfStats, *perfTestUtils.Config, perfTestUtils.FileSystem, string, string),
```

Comment:
Added string parameter to pass testSuite.TestStrategy to the report template to show/hide conditional data. See line 281.